src/app/dashboard/competitions/[competitionId]/register-form/edit/components/EditCompetitionRegisterForm.tsx

```diff
@@ -0,0 +1,254 @@
+            </a>
+          )}
+          <FileInput
+            label="Cambiar bases de la competencia"
+            name={fields.competitionBases.name}
+            accept=".pdf"
+            errors={fields.competitionBases.errors}
+          />
+
+          <p className="text-lg text-gray-500">Datos del participante</p>
+
+          <input
+            type="hidden"
+            name={fields.registerFields.name}
+            value={JSON.stringify(
+              newFieldName.map((fieldName, index) => ({
+                name: fieldName,
+                type: newFieldType[index],
+              })),
+            )}
+          />
+          {fields.registerFields.errors?.[0] && (
+            <p className="text-error">{fields.registerFields.errors?.[0]}</p>
+          )}
+
+          {newFieldName.length > 0 && newFieldName.map((fieldName, index) => (
+            <div key={index} className="flex flex-row items-center gap-2">
+              <p className="flex-none">- {fieldName}:</p>
+              <p className="flex-none text-gray-500">{newFieldType[index]}</p>
+              <button
+                type="button"
+                className="btn btn-ghost btn-sm text-error"
+                onClick={() => handleDeleteField(index)}
+              >
+                <Trash className="size-5" />
+              </button>
+            </div>
+          ))}
+          <FormSelect
+            label="Identificador del participante"
+            name={fields.participantIdentifier.name}
+            options={participantIdentifierOptions}
+            placeholder="Selecciona un identificador del participante"
+            required={false}
+            errors={fields.participantIdentifier.errors}
+            value={participantIdentifier}
+            onChange={(value) => setParticipantIdentifier(value)}
+          />
+          <div className="flex flex-row items-end gap-4">
+            <FormInput
+              label="Nombre del campo"
+              name="currentFieldName"
+              type="text"
+              value={currentFieldName}
+              onChange={(e) => setCurrentFieldName(e.target.value)}
+            />
+
+            <FormSelect
+              label="Tipo de campo"
+              name="currentFieldType"
+              options={fieldTypeOptions.filter((type) => type !== participantIdentifier)}
+              placeholder="Selecciona un tipo de campo"
+              required={false}
+              errors={undefined}
+              value={currentFieldType}
+              onChange={(value) => setCurrentFieldType(value)}
+            />
+
+            <button 
+              className="btn btn-primary" 
+              onClick={handleAddField}
+              disabled={!currentFieldName || !currentFieldType}
+            >
+              Agregar campo
+            </button>
+          </div>
+
+          <SubmitButton
+            label="Editar formulario de registro"
+            loadingLabel="Editando..."
```

Comment:
Recomiendo:
```suggestion
            label="Guardar Cambios"
            loadingLabel="Guardando..."
```
Así mantenemos consistencia con el formulario de edición de competencia